lib/DDG/Spice/Statista.pm

```diff
@@ -0,0 +1,26 @@
+package DDG::Spice::Statista;
+# ABSTRACT: Returns statistics from statista.com.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "statistics about soccer";
+description "Shows example statistics";
+name "Statista";
+category "facts";
+
+my $limit = 16;
+my $lang = 2;
+
+triggers startend => "statistic", "statistics", "stats";
```

Comment:
After some testing, I feel that this will be too broad of a trigger.

Let's change this to:

```perl
triggers start => "statistics on", "statistics of", "statistics about", "statistics about", ...
```

Alternatively you could `handle query_lc` and then use a regex to make sure the query begins with `/stat(istic)?s? (on|of|for|about) (.+)/` or ends in the word "statistics", `/(.+) stat(istic)?s?$/`